examples/ipfs-from-ts/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "ipfs-from-ts",
+  "private": true,
+  "dependencies": {
+    "ipfs": "^0.50.1"
+  },
+  "scripts": {
+    "test": "npx typescript --noEmit"
```

Comment:
1. Can you add `typescript` as a dev dep instead of using npx otherwise we'll get whatever the latest version is at runtime which makes our tests non-deterministic.
2. Don't we need to pass a file to the `typescript` command?